pages/blog/2022Q1-summary.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Summary of things that happened around the AsyncAPI Initiative until end of April 2022"
+date: 2022-04-28T06:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+cover: /img/posts/2022Q1-summary/cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Gardner
+excerpt: 'New AsyncAPI brand. AsyncAPI 2.4 and work on 3.0 spec release. Meetings organization changes. Conference organization. AND SO MUCH MORE!'
+featured: true
+---
+
+> TODO here will be link to podcast for those that do not read articles
+
+Welcome to another update for AsyncAPI Initiative. In this article, I want to highlight the most important things around AsyncAPI in the first few months of 2022.
+
+I usually ask the community if folks have anything important that I should put in my updates. Still, it is a pretty much subjective article based mainly on things I see in my personal bubble :smiley:
```

Comment:
```suggestion
I usually ask the community if folks have anything important that I should put in my updates. Still, this is a pretty subjective article because it's based mainly on things I see in my personal bubble. :smiley:
```